sdk/anomalydetector/azure-ai-anomalydetector/samples/sample_detect_change_point.py

```diff
@@ -22,9 +22,11 @@
 """
 
 import os
+import pandas as pd
+
 from azure.ai.anomalydetector import AnomalyDetectorClient
 from azure.core.credentials import AzureKeyCredential
-import pandas as pd
+from azure.ai.anomalydetector.models import *
```

Comment:
@Xingyu-Yue You can `pip install black` and `black samples/` to format the code automatically